File: Python/1015.py
import math
x = input().split()
y = input().split()

x1 = float(x[0])
y1 = float(x[1])
x2 = float(y[0])
y2 = float(y[1])

dist = math.sqrt((x2 - x1)**2 + (y2 - y1)**2)
print(f'{dist:.4f}')

File: C/1015.c
#include <stdio.h>
#include <math.h>

int main() {

    float x1,y1,x2,y2,d1,d2,dT;
    scanf("%f%f", &x1,&y1);
    scanf("%f%f", &x2,&y2);

    d1 = x2 - x1;
    d1 = d1 * d1;
    d2 = y2 - y1;
    d2 = d2 * d2;
    dT = d1 + d2;
    dT = sqrt(dT);

    printf("%.4f\n", dT);
}
